crates/cranelift/src/debug/transform/debug_transform_logging.rs

```diff
@@ -123,7 +123,10 @@ impl<'a> fmt::Debug for DieDetailedSummary<'a> {
                 AttributeValue::CallingConvention(value) => write!(f, "{value}"),
                 AttributeValue::Inline(value) => write!(f, "{value}"),
                 AttributeValue::Ordering(value) => write!(f, "{value}"),
-                AttributeValue::UnitRef(offset) => write!(f, "0x{:08x}", offset.0),
+                AttributeValue::UnitRef(offset) => {
+                    let section_offset = offset.to_unit_section_offset(unit);
+                    write!(f, "0x{:08x}", get_offset_value(section_offset))
+                }
```

Comment:
Just calling out this change as unrelated - we can merge it in this PR. I don’t understand cranelift well enough to review it properly but since you seem to know this stuff inside out I’m fine with it.

Reply:
Thanks!

This is a bugfix for the debug logging you can turn on to see how DIEs get processed - it now prints the section (global) offsets for all referenced DIEs, allowing one to easily search for them in the editor.